pkg/osinfo/layer_scanner.go

```diff
@@ -117,12 +116,26 @@ func (loss *LayerScanner) extractFileFromTar(tarPath, filePath, destPath string)
 	}
 	defer f.Close()
 
+	// Read the first bytes to determine if the file is compressed
+	var sample [2]byte
+	var compressed bool
```

Comment:
we are looking for gzip magic bytes, so `gzipped` may be better?

Reply:
Renamed the var :+1: